deploy/grafana-dashboard/poseidon.dashboard.py

```diff
@@ -0,0 +1,25 @@
+from grafanalib.core import Dashboard, Templating, Time
+
+from availability_row import availability_row
+from general_row import general_row
+from runner_insights_row import runner_insights_row
+from variables import stage_variable, environment_variable
+
+dashboard = Dashboard(
+    title="Poseidon autogen",
+    timezone="browser",
+    panels=[
+        general_row,
+        runner_insights_row,
+        availability_row
+    ],
+    templating=Templating(list=[
+        stage_variable,
+        environment_variable
```

Comment:
Something seems different here to our current dashboard:

- Previously, we also allowed to filter for status code, which only seems useful to some graphs
![Bildschirm­foto 2022-10-27 um 00 01 19](https://user-images.githubusercontent.com/7300329/198148537-fe9b71b7-ab81-4888-8ee0-1d036fe10525.png)

- Stage is fixed to production instead of allowing other environments as well?
![Bildschirm­foto 2022-10-27 um 00 01 00](https://user-images.githubusercontent.com/7300329/198148800-df24ec82-525c-4636-8760-cf57f72ce939.png)![Bildschirm­foto 2022-10-27 um 00 01 04](https://user-images.githubusercontent.com/7300329/198148802-a81d3776-bfc0-428c-b047-5c80bb63a113.png)

- Environment ID is not showing all IDs (similar to the graph "Prewarming Pool Size") and there is a "null" entry
![Bildschirm­foto 2022-10-27 um 00 01 08](https://user-images.githubusercontent.com/7300329/198148842-3bb32b84-019e-4d84-a403-93d4b7774966.png)![Bildschirm­foto 2022-10-27 um 00 01 12](https://user-images.githubusercontent.com/7300329/198148845-2c66cc31-41d2-470f-b75c-17c248cbe08f.png)

Reply:
> * Previously, we also allowed to filter for status code, which only seems useful to some graphs

Exactly, the filter for the status code has increased the complexity but had a very limited usage, so I removed it. Are you fine with that?

> Stage is fixed to production instead of allowing other environments as well?

I had falsely just queried the stages of environments that have been active in the selected time range. Now we display all stages of the last year (as in the original dashboard).

> Environment ID is not showing all IDs (similar to the graph "Prewarming Pool Size") and there is a "null" entry

See https://github.com/openHPI/poseidon/pull/200#discussion_r1008734037

Reply:
>> Previously, we also allowed to filter for status code, which only seems useful to some graphs
> 
> Exactly, the filter for the status code has increased the complexity but had a very limited usage, so I removed it. Are you fine with that?

Yes, that's absolutely fine for me 👍  Thanks for taking care!